warehouse/warehouse.go

```diff
@@ -327,8 +328,17 @@ func (wh *HandleT) backendConfigSubscriber() {
 					if destination.DestinationDefinition.Name == wh.destType {
 						namespace := wh.getNamespace(destination.Config, source, destination, wh.destType)
 						warehouse := warehouseutils.WarehouseT{Source: source, Destination: destination, Namespace: namespace}
+						var cronExpression string
+						if val, ok := destination.Config["cronExpression"].(string); ok {
+							cronExpression = val
+						}
+						fmt.Println(cronExpression)
+						if warehousecron.IsCronExpressionPresent(cronExpression, connectionString(warehouse)) {
```

Comment:
Rename it to IsValidCron(). Along with empty string check, also see if it is a valid cron expression.